### ISSUE TITLE
Any plans to support System.Drawing.Common?

### ISSUE THREAD

**Issue by Lupusa87** [aspnet/Blazor#1149]
Maybe it is not so important to have this support on early development phase, but it's interesting if it can be supported and how high importance will it have.

It is based on mono, so maybe can be easy implemented.

https://www.nuget.org/packages/System.Drawing.Common
https://www.nuget.org/packages/Microsoft.Windows.Compatibility
https://github.com/CoreCompat/CoreCompat

![untitled](https://user-images.githubusercontent.com/40779557/42899160-2852c026-8a93-11e8-9425-ce7b5c7ec947.png)

**Comment by SteveSandersonMS** [comment]
It’s probably best to post this question to the Mono repo, as that’s where this would be implemented. Thanks!

**Comment by StevenTCramer** [comment]
They have this page.

https://www.mono-project.com/docs/gui/drawing/

Which makes me think it would work but it doesn't.  

I was trying to use QRCoder and found this error.

**Comment by Lupusa87** [comment]
I was not able to use it and leaved this.
I was using canvas (fully depended on js) for drawing purposes and then swithed to svg because it can be drawed from blazor without js interop.
I never touched QRCoder.
If canvas will be good choice for you there is blazor extention for canvas with is doing all js stuff for you.
If svg will be good fit you can use my library from nuget BlazorSvgHelper with will help you manipulate svg easier.
For future investigation on this issue it will be better to ask mono folks on their repo as Steve recomended.

**Comment by Andrzej-W** [comment]
@Lupusa87 By the way, is this your code in index.html:
```js
window.location.href = "https://lupblazordemos.z13.web.core.windows.net/";
```
Maybe that is the source of problems in antivirus.

**Comment by Lupusa87** [comment]
Thank you. Yes I do redirection because fist I was using free azure web app and shared this site in many places, then because free option had limitations site often was stopped, I hosted my app as static website in azure storage and did redirection to keep original site address working.
Raw html renderer I tryied befor blazor got built-in feature, it is smal html parser and renderer, I have there code to block any script in input.
All app is on github and we can investigate why you have security warnings.
I don't do any dangerous code on purpose :)
Only one I can have doubt is that I am using js eval() in canvas clock, as I know it can be security problem and maybe antivirus reports it. But actually eval can be problem for me someone not to hack my app and not vice versa I to hack my client.
